operator/keys/rsa_linux.go

```diff
@@ -26,30 +28,62 @@ type publicKey struct {
 }
 
 func init() {
-	// TODO: check multiple versions of openssl
 	// TODO: fallback to stdlib when openssl is not available
-	if err := openssl.Init(); err != nil {
+	version := getVersion()
+	if err := openssl.Init(version); err != nil {
 		panic(err)
 	}
 }
 
+// getVersion returns the highest available OpenSSL version.
+func getVersion() string {
```

Comment:
Can you please add a comment that this code is taken from https://github.com/golang-fips/openssl/blob/v2/openssl_test.go?

Reply:
Yup, the reason is https://github.com/golang-fips/openssl doesn't really provide any examples/docs on how this should be done, so I found this approach to work for me (in Docker),

added a note on this.